spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -122,6 +122,45 @@
 	 */
 	private boolean useSpringApplicationJson = true;
 
+	private PortBound portBound = new PortBound();
```

Comment:
These feels a bit awkward as is:
```
spring.cloud.deployer.local.port-bound.upper
spring.cloud.deployer.local.port-bound.lower
```
would it be better to rename to:
```
spring.cloud.deployer.local.port-range.high
spring.cloud.deployer.local.port-range.low
```

Reply:
agree